docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
 It may be unstable and documentation is limited.
 If you have any questions, please contact us at [hello@open-diffix.org](mailto:hello@open-diffix.org).
 
-# User Guide
+# Extension configuration
 
-This document provides detailed information about the configuration, behavior and recommended usage
-of __Diffix for PostgreSQL__.
+This document provides detailed information about the configuration, behavior and recommended usage of this extension __pg_diffix__.
 
-## What is Diffix?
+## Background reading
 
-Diffix is a bundled set of mechanisms for anonymizing structured data. It was jointly developed by Aircloak GmbH and
-the Max Planck Institute for Software Systems. Diffix exploits mechanisms that have been in use by national statistics
-offices for decades: aggregation, generalization, noise, suppression, and swapping. It automatically applies these
-mechanisms as needed on a query-by-query basis to minimize noise while ensuring strong anonymity.
+The [Open Diffix website](https://www.open-diffix.org) has general information about the Open Diffix project. The [website FAQ](https://www.open-diffix.org/faq) is a good starting point. [This article](https://www.open-diffix.org/blog/diffix-elm-automates-what-statistics-offices-have-been-doing-for-decades) is a high-level overview of the anonymization mechanisms used by Diffix. [This paper](https://arxiv.org/abs/2201.04351) is a complete specification and privacy analysis of Diffix (version Elm).
 
-## What is Diffix for PostgreSQL?
+## Configuration objects
 
-__Diffix for PostgreSQL__ is an implementation of the Diffix mechanism for PostgreSQL databases.
-It is provided as an extension module for PostgreSQL version 13 or higher.
+Extension behavior is controlled by `security labels` and `settings`. Security labels specific to this extension are assigned to tables, columns, and roles (users). Settings are general parameters associated with system operation. Settings can be assigned in the configuration file and are instantiated per session.
```

Comment:
Instead of marking "`security labels` and `settings`" with code style I would use italics or bold.